app/controllers/stripe_payments/webhooks_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module StripePayments
+  class WebhooksController < ApplicationController
+    skip_before_action :verify_authenticity_token
+
+    # rubocop:disable Metrics/AbcSize
```

Comment:
We can add this file to Rubocop exceptions instead of that.

Reply:
Not exactly, because other things are checked by rubocop. I just wanted to omit `Metrics/AbcSize` here.